### ISSUE TITLE
Change the base font size for Treemap 

### ISSUE THREAD

**Issue by fguimara** [apexcharts/apexcharts.js#1983]
How to make the label font size base occupy a larger proportion of the related rectangle on the treemap?

**Comment by antiden** [comment]
Need.

**Comment by antiden** [comment]
ok, now its working.

1. Need add parent class for chart. Example: `<VueApexCharts height="350" :options="chart.options" :series="chart.series" :class="'chart-' + chart.type_id"></VueApexCharts>` where class is .chart-16
2. Add css:
```
.chart-16
  overflow hidden
  
  text
    font-size 14px !important
```

Ready. Now fonts have one size.

**Comment by Ashesh3** [comment]
> ok, now its working.
> 
> 1. Need add parent class for chart. Example: `<VueApexCharts height="350" :options="chart.options" :series="chart.series" :class="'chart-' + chart.type_id"></VueApexCharts>` where class is .chart-16
> 2. Add css:
> 
> ```
> .chart-16
>   overflow hidden
>   
>   text
>     font-size 14px !important
> ```
> 
> Ready. Now fonts have one size.

Does some text not go out of the rectangles if we do this?

**Comment by antiden** [comment]
@Ashesh3 

No, i added for parent block, overflow hidden. Text will be trim

**Comment by trannel** [comment]
Hey, @antiden I tried your solution but couldn't get it to work 100%. Does it still work for you? I'm using react-apexcharts, so that could also be the reason. I was able to set the font-size using
```CSS 
.treemap .apexcharts-datalabel {
  font-size: 14px !important;
}
```
, while setting the `treemap` class on one of the parent elements of the treemap. I could not get rid of the overflow. I tried setting `overflow: hidden` and similar attributes on the treemap, one of its ancestors, the element with the class `.apexcharts-datalabel`, and its 2 ancestors.

**Comment by trannel** [comment]
Hey @antiden, I tried your solution but couldn't get it to work. Does it still work for you? I'm using react-apexcharts, so that could also be an issue. I was only able to set the font-size using
```CSS
.treemap .apexcharts-datalabel {
  font-size: 14px !important;
}
```
, while setting `treemap` as class of one of the ancestors of the treemap. I tried setting `overflow: hidden` and similar attributes to the treemap itself, the ancestor with the class `treemap`, the element with the class `apexcharts-datalabel`, and its 2 ancestors. The overflow always stayed visible.

**Comment by Adrian-GP** [comment]
This is still an issue. Even if you set the datalabels manually, the treemap's font size won't change.

```
 "dataLabels" : {
            "style": {
                "fontSize": "16px"
            }
        },

```

**Comment by antonmelbardis** [comment]
any update on this ?

**Comment by oSamper** [comment]
is this still an issue? Any update on this?
